zebra-network/src/peer/client/tests.rs

```diff
@@ -90,10 +90,12 @@ impl ClientTestHarness {
 
     /// Drops the mocked heartbeat shutdown receiver endpoint.
     pub fn drop_heartbeat_shutdown_receiver(&mut self) {
-        let _ = self
```

Comment:
Did this trip a linter warning?

Reply:
Yes, a future was implicitly dropped without awaiting it.